client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -94,13 +95,22 @@ const useInvestigationTable = (): useInvestigationTableOutcome => {
       });
   }, [user.id, classes.errorAlertTitle]);
 
-
-  const onInvestigationRowClick = (epidemiologyNumber: number) => {
+  const onInvestigationRowClick = (epidemiologyNumberVal: number) => {
```

Comment:
why adding the val?

Reply:
because i wont is to take the function val and not the store val

Reply:
תיקנתי